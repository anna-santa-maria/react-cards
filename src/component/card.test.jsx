import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Card from './card';
import GameModel from '../model/game_model';

configure({ adapter: new Adapter() });

describe('<Card/>', () => {
    const [{ type, rank }] = GameModel.generate(4);

    const props = {
        type,
        rank
    };

    describe('render', () => {
        it('with default/required props', () => {
            const c = shallow(<Card {...props} />);

            expect(c).toMatchSnapshot();
        });
    });
});
