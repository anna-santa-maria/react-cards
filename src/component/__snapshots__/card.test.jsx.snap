// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card/> render with default/required props 1`] = `
<span
  className="card hearts rank14"
/>
`;
